relayer/chains/substrate/key_test.go

```diff
@@ -14,10 +15,11 @@ func TestKeyRestoreAndRetrieve(t *testing.T) {
 	mnemonic := "blind master acoustic speak victory lend kiss grab glad help demand hood roast zone lend sponsor level cheap truck kingdom apology token hover reunion"
 	expectedAddress := "5Hn67YZ75F3XrHiJAtiscJMGQ4zFNw9e45CNfLuxL6vEVYz8"
 
-	provider, err := getTestProvider()
-	require.Nil(t, err)
+	var provider = substrate.SubstrateProvider{}
```

Comment:
`provider := substrate.SubstrateProvider{}` is slightly less verbose and achieves the same thing